crates/forge-runner/src/printing.rs

```diff
@@ -61,7 +61,15 @@ fn format_detailed_resources(
     used_resources: &UsedResources,
     tracked_resource: ForgeTrackedResource,
 ) -> String {
-    let sorted_syscalls = sort_by_value(&used_resources.syscall_counter);
+    // Sort syscalls by call count
+    let mut sorted_syscalls: Vec<_> = used_resources
+        .syscall_usage
+        .clone()
+        .into_iter()
```

Comment:
nit:
```suggestion
        .iter()
```

Reply:
By the way, there is also `.cloned()` which clones the items from the iterator.